.github/workflows/e2e-ci.yaml

```diff
@@ -2,23 +2,42 @@ name: LoRa E2E CI
 concurrency: e2e
 on: # rebuild any PRs and main branch changes
   schedule:
-    - cron: '59 23 * * *'
+    - cron: '59 23 * * 1,2,3,4,5'
   workflow_dispatch:
     inputs:
       RunE2ETestsOnly:
         description: 'Run E2E tests only'
         default: false
       TestsToRun:
-        default: '[\"SensorDecodingTest\",\"OTAAJoinTest\",\"ABPTest\",\"OTAATest\",\"MacTest\",\"ClassCTest\",\"C2DMessageTest\",\"MultiGatewayTest\"]'
+        default: '[SensorDecodingTest,OTAAJoinTest,ABPTest,OTAATest,MacTest,ClassCTest,C2DMessageTest,MultiGatewayTest]'
         description: 'tests to run'
       TxPower:
         description: 'TXPower value to use in E2E tests'
         default: '6'
 env:
-  buildConfiguration: 'Release'
-  AZURE_FUNCTIONAPP_NAME: loramoduleintegrationtest
+  BUILD_CONFIGURATION: 'Release'
   TESTS_FOLDER: Tests
   TESTS_RESULTS_FOLDER: Tests/TestResults
+  # these environments variable are for the deployment to IoT Edge
+  CONTAINER_REGISTRY_ADDRESS: "${{ secrets.CI_ACR_NAME }}.azurecr.io"
+  CONTAINER_REGISTRY_USERNAME: "${{ secrets.CI_ACR_NAME }}"
+  CONTAINER_REGISTRY_PASSWORD: ${{ secrets.CI_ACR_CREDENTIALS }}
+  IOT_EDGE_DEPLOYMENT_ID: ""
+  NET_SRV_LOG_LEVEL: 1
+  NET_SRV_LOGTO_HUB: false
+  NET_SRV_LOGTO_UDP: true
+  NET_SRV_IOTEDGE_TIMEOUT: 0
+  NET_SRV_LOG_TO_UDP_ADDRESS: "itestup"
```

Comment:
Should this be fixed?

Reply:
Is this comment in the scope of the migration to TCP? I am probably going to merge first as Atif wants to run a full ci on hist branch @atifaziz